lib/tasks/compare_xml_collections.rake

```diff
@@ -0,0 +1,121 @@
+require 'libxml_to_hash'
+
+namespace :collection do
+  desc 'Translate a collection from native format to UMM JSON and back to native format'
```

Comment:
I would be concerned about this somewhat complicated task not having tests. But since you are going to be adding the functionality into the app I expect there to be tests added then.

Reply:
correct.